kafka/consumer/group.py

```diff
@@ -567,18 +564,34 @@ def topics(self):
         future = cluster.request_update()
         self._client.poll(future=future)
         cluster.need_all_topic_metadata = stash
+
+    def topics(self):
+        """Get all topics the user is authorized to view.
```

Comment:
Can you add:
`This will always issue a remote call to the cluster to fetch the latest information`.